sled-agent/src/bin/sled-agent.rs

```diff
@@ -117,16 +119,18 @@ async fn do_run() -> Result<(), CmdError> {
             //
             // This should remain equivalent to the HTTP request which can
             // be invoked by Wicket.
-            if let Some(rss_config) = rss_config {
-                match server.agent().start_rack_initialize(rss_config) {
-                    // If the rack has already been initialized, we shouldn't
-                    // abandon the server.
-                    Ok(_)
-                    | Err(
-                        bootstrap_agent::RssAccessError::AlreadyInitialized,
-                    ) => {}
-                    Err(e) => {
-                        return Err(CmdError::Failure(e.to_string()));
+            if rss_initiator {
```

Comment:
Explicitly calling this change out. For multi-node configs, we now specify in config who the RSS initiator is.

Reply:
What about the previous check for if the relevant `config-rss.toml` path? For multi-node configs, this is equivalent to making sure only one node has that file.

Reply:
I think that could probably work. I'll test it out on my next lap.

Reply:
I verified this is not needed and have removed this config option. Thanks!